.circleci/config.yml

```diff
@@ -19,3 +19,31 @@ jobs:
       - run: go get github.com/Masterminds/glide
       - run: glide install --strip-vendor
       - run: make build_linux
+
+  docker_push_master:
+    working_directory: /go/src/github.com/aerogear/aerogear-app-metrics
```

Comment:
The wonders of copy + paste. This needs to be changed :)

Reply:
:smile_cat: